### PR TITLE
Fix: Sentry errors and UI Fixes

### PR DESCRIPTION
**Pull request by mul53**
- Add balance on mobile view
- Handle sentry errors better

### CODE REVIEW

File: src/hooks/useApproveCallback.ts [hunk 1]
```diff
@@ -91,12 +91,14 @@ export function useApproveCallback(
...
         })
       })
-      .catch((error: Error) => {
-        Sentry.captureException(error, {
-          tags: {
-            action: 'Approve Token'
-          }
-        })
+      .catch((error: any) => {
+        if (error?.code !== 4001) {
```
Comment (leonprou) [inline src/hooks/useApproveCallback.ts:95]: can you add explanations about the error status?

File: src/state/bridge/hooks.ts [hunk 2]
```diff
@@ -148,28 +148,36 @@ export function useBridgeActionHandlers(): {
...
   return useAsyncMemo(async () => {
-    if (!chainId || !account || !library || !tokenAddress || !isMultiErc20ToErc677BridgeToken(tokenAddress)) return
+    if (!isHome || !account || !library || !tokenAddress || !isMultiErc20ToErc677BridgeToken(tokenAddress)) return
 
     const address = getHomeMultiErc20ToErc677BridgeAddress()
     const contract = getHomeMultiAMBErc20ToErc677Contract(address, library, account)
     const fee = await contract.getFee(HOME_TO_FOREIGN_FEE_TYPE_HASH, tokenAddress)
     return formatEther(fee)
-  }, [account, chainId, library, tokenAddress])
+  }, [isHome, account, chainId, library, tokenAddress])
```
Comment (leonprou) [inline src/state/bridge/hooks.ts:160]: I think `chainId ` is not used here. Can  you go over the linting issues? Fix or put a ignore when needed

**Review by leonprou — CHANGES_REQUESTED** [review]
Looks good.
Please see the comments

File: src/state/bridge/hooks.ts (hunk 2 above)
Comment (mul53) [reply src/state/bridge/hooks.ts]: I have removed chainId from the functions

File: src/hooks/useApproveCallback.ts (hunk 1 above)
Comment (mul53) [reply src/hooks/useApproveCallback.ts:96]: Added a comment to the code, usually 4001 code is from an error when the user rejected the transaction

**Review by leonprou — APPROVED** [review]

**Pull request merged**